### ISSUE TITLE
Model Inspector: nndct_relu can't be assigned to DPU

### ISSUE THREAD

**Issue by brtsfr** [Xilinx/Vitis-AI#1501]
hi everyone, happy holiday! i got an issue and please take a look. 
i am learning Vitis-AI to deploy a simple cnn model on KV260. I run Vitis-AI 3.5 in pytorch-cpu docker on ubuntu20.04. 
According to Vitis-AI 3.5 doc, i run a Model_Inspector.py to check my model. And i got the note. Here attached all the relative files. Thanks in advance.
[Model_Inspector.py.txt](https://github.com/user-attachments/files/18249774/Model_Inspector.py.txt)
[screenlog_ModelInspector_Run.txt](https://github.com/user-attachments/files/18249741/screenlog_ModelInspector_Run.txt)
[inspect_0x101000016010406.txt](https://github.com/user-attachments/files/18249742/inspect_0x101000016010406.txt)
![inspect_0x101000016010406](https://github.com/user-attachments/assets/a6ca89fd-e2f1-47e9-8ead-fafe8ff996de)

**Comment by brtsfr** [comment]
ok, i found the solution on xilinx forum. it is a known issue. a batchnorm should be added between maxpool2d and relu.
[https://adaptivesupport.amd.com/s/article/000037197?topicId=0TO2E000000YKY9WAO&topicName=Vitis%20AI%20&%20AI](url)
cheers
